app/controllers/thread_controller.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
```

Comment:
same from this [comment](https://github.com/floresjann/DietCake-Board/pull/1/files#r122352853)